db/migrate/20170509210708_add_use_notifications_to_info_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseNotificationsToInfoRequest < ActiveRecord::Migration
+  def change
+    add_column :info_requests, :use_notifications, :boolean
```

Comment:
I had that initially, but it means that we can't detect the difference between someone *forcing* it to false and rails initialising it with the database default, which in turn means we can't tell [whether to apply our own initialisation or not](https://github.com/mysociety/alaveteli/pull/3975/files#diff-5f7dfe1185cac97cbc9f9dc356abce7cR1636).